healthcareai/tests/test_db_validation.py

```diff
@@ -1,7 +1,7 @@
 import unittest
```

Comment:
1. Good!
2. I renamed it for consistency - make sure to `git pull` to grab the latest code
3. Ah! so, you may have to run a chunk of SQL listed in getting started to create the table on your local MSSQL server.